code/_onclick/hud/guardian_hud.dm

```diff
@@ -53,6 +53,9 @@
 		if(istype(summoner_loc, /obj/machinery/atmospherics))
 			to_chat(G, "<span class='warning'>You can not manifest while in these pipes!</span>")
 			return
+		if(istype(summoner_loc, /obj/structure/closet/cardboard/agent))
+			to_chat(G, "<span class='warning'>You can not manifest while your Stealth Implant is active!</span>")
```

Comment:
This kinda feels like it implies the holopara has the implant rather than the host, might sound better as
 "you can not manifest while _the_ stealth implant is active!"